convert_torcx_image.sh

```diff
@@ -32,6 +32,12 @@ for FILE in "${SYSEXTNAME}"/usr/lib/systemd/system/*.service "${SYSEXTNAME}"/usr
       -e 's,After=torcx.target,,g' -e 's,Requires=torcx.target,,g' "${FILE}"
   fi
 done
+for ENTRY in "${SYSEXTNAME}/usr/lib/systemd/system/"*.wants/*; do
+  UNIT=$(echo "${ENTRY}" | rev | cut -d / -f 1 |rev)
+  TARGET=$(echo "${ENTRY}" | rev | cut -d / -f 2 |rev | sed 's/.wants$//g')
```

Comment:
This could be basename of dirname, so `TARGET=$(basename "$(dirname "${ENTRY}")" | sed 's/\.wants$//'), but it's up to you - it's not that much simpler than what you wrote. So suggesting just some minor things:

```suggestion
  TARGET=$(echo "${ENTRY}" | rev | cut -d / -f 2 | rev | sed 's/.wants$//')
```